### DISCUSSION TITLE
prices from MFAPI are not converted to default currency

### DISCUSSION THREAD

**Discussion by kunaguvarun** [ananthakumaran/paisa#239] [Ideas]
Hi,

This is in continuation to my previous issue which is moved to discussions now but I think its an issue and worth your attention. Prices fetched from Yahoo are converted from INR to default currency which is USD in my case. However, prices from MFAPI are retained in INR and not converted to USD.

![image](https://github.com/ananthakumaran/paisa/assets/3099125/c757b68c-2401-4810-b13a-57c7b0773985)
![image](https://github.com/ananthakumaran/paisa/assets/3099125/f4129b21-ef95-4b10-9258-cd65d516ff9d)
![image](https://github.com/ananthakumaran/paisa/assets/3099125/f4b37f2a-4e43-4b66-aea8-0cb994e6b2a3)
![image](https://github.com/ananthakumaran/paisa/assets/3099125/b653d1af-b396-44d3-8c0e-e89436da1159)

**Doing the same with MFAPI now**

INR is retained as INR and nit converted to USD thereby messing up balances

![image](https://github.com/ananthakumaran/paisa/assets/3099125/4bd042a8-a23f-451c-9d83-3005d1c0e23a)
![image](https://github.com/ananthakumaran/paisa/assets/3099125/68ceae9d-89b0-4fe3-a006-1521c88dbd10)

**Comment by ananthakumaran** [comment]
If a price provider supports currency conversion, paisa automatically converts to the default currency. This is the case with yahoo and alphavantage.

I understand handling this scenario for all price providers would be better, but I would consider that a feature request. 

I use discussions for all feature requests, moving this to discussions as well.
